CHANGELOG.md

```diff
@@ -3,7 +3,15 @@
 
 ## Unreleased
 
-**Note:** As of this release, the minimum supported version of NodeJS is 14.x.
+**Note:** As of this release, the minimum supported version of NodeJS is **14.x**.
+
+### Add
+
+- Support for converting signed payloads ([CAP-40](https://stellar.org/protocol/cap-40)) to and from their StrKey (`P...`) representation ([#511](https://github.com/stellar/js-stellar-base/pull/511)).
+
+- Support for creating transactions with the new preconditions ([CAP-21](https://stellar.org/protocol/cap-21)) via `TransactionBuilder` ([#513](https://github.com/stellar/js-stellar-base/pull/513)).
+
+- A way to convert between addresses (like `G...` and `P...`, i.e. the `StrKey` class) and their respective signer keys (i.e. `xdr.SignerKey`s), particularly for use in the new transaction preconditions ([#520](https://github.com/stellar/js-stellar-base/pull/520)).
```

Comment:
There's probably a few more things we've added here? like api support for protocol 19, etc.

Reply:
API support is part of stellar/js-stellar-sdk; I don't want to conflate what changed where